src/components/expanded-state/SwapDetailsState.js

```diff
@@ -41,13 +41,11 @@ const SwapDetailsState = ({
   inputCurrencySymbol,
   inputExecutionRate,
   inputNativePrice,
-  onRefocusInput,
   outputCurrencySymbol,
   outputExecutionRate,
   outputNativePrice,
 }) => {
   const { goBack } = useNavigation();
-  useEffect(() => () => onRefocusInput(), [onRefocusInput]);
```

Comment:
If you open the SwapDetailsState and then close it, the input does not refocus on swap

Reply:
RE: renaming 

I think those are two different things:
- onRefocusInput is / was used for the input specifically
- restoreFocusOnSwapModal : sets the focus on the screen at the navigator level